### PR TITLE
ZFIN-9354: Fix hibernate issues on tomcat10 branch 

### PR DESCRIPTION
**Pull request by rtaylorzfin**

### CODE REVIEW

File: source/org/zfin/expression/presentation/FishExpressionSummaryController.java [hunk 1]
```diff
@@ -92,36 +92,26 @@ protected String getFishExpressionFigureSummaryStandard(@RequestParam String fis
...
-        model.addAttribute("expressionCriteria", expressionCriteria);
-        List<FigureSummaryDisplay> figureSummaryDisplayList = FigureService.createExpressionFigureSummary(expressionCriteria);
-        model.addAttribute("figureSummaryDisplayList", figureSummaryDisplayList);
-
-        model.addAttribute(LookupStrings.DYNAMIC_TITLE, fish.getName() + " Expression Figure Summary");
-        return "expression/genotype-figure-summary";
-
-    }
 
+    //Not seeing any traffic to this page in our access logs other than some hits from bingbot
```
Comment (cmpich) [inline source/org/zfin/expression/presentation/FishExpressionSummaryController.java:97]: Yes, remove the method if it is not being called from within our app.

Comment (rtaylorzfin) [reply source/org/zfin/expression/presentation/FishExpressionSummaryController.java]: I removed it

File: source/org/zfin/expression/repository/HibernateExpressionRepository.java [hunk 2]
```diff
@@ -955,54 +989,48 @@ public List<ExpressionFigureStage> getExperimentFigureStagesByGeneAndFish(String
...
-
-//
-// TODO (ZFIN-9354): hibernate migration change, need to implement ordering in the query
-//        hql += """
-//                order by efs.figure.orderingLabel, gene.abbreviationOrder
-//                         , fish.name
-//                         , efs.expressionExperiment.assay.displayOrder
-//                         , efs.startStage.abbreviation
-//            """;
+                    SELECT DISTINCT efs FROM ExpressionFigureStage AS efs
```
Comment (rtaylorzfin) [inline source/org/zfin/expression/repository/HibernateExpressionRepository.java:998]: Are there efficiency concerns here?

Comment (rtaylorzfin) [reply source/org/zfin/expression/repository/HibernateExpressionRepository.java:998]: create a ticket to revisit

File: source/org/zfin/expression/repository/HibernateExpressionRepository.java [hunk 3]
```diff
@@ -1011,126 +1039,40 @@ public ExpressionFigureStage getExpressionFigureStage(Long id) {
...
-            hql += "           and gene.zdbID = :geneID ";
-        }
-        if (fishID != null) {
-            hql += "           and fishox.fish.zdbID = :fishID ";
-        }
-        hql += "    order by gene.abbreviationOrder, " +
-               "             fishox.fish.name, " +
-               "             fishox.experiment.name, " +
-               "             experiment.assay.displayOrder ";
+        String hql = """
```
Comment (rtaylorzfin) [inline source/org/zfin/expression/repository/HibernateExpressionRepository.java:1042]: include in ticket

File: source/org/zfin/mutant.hbm.xml [hunk 4]
```diff
@@ -145,7 +145,7 @@
         </set>
 
         <property name="flag" column="mrkrgoev_gflag_name" not-null="false" type="goFlag"/>
-        <many-to-one name="qualifierRelation" column="mrkrgoev_relation_term_zdb_id" not-null="true"/>
+        <many-to-one name="qualifierRelation" column="mrkrgoev_relation_term_zdb_id"/>
```
Comment (rtaylorzfin) [inline source/org/zfin/mutant.hbm.xml:148]: Are there other instances like this?

File: test/org/zfin/mapping/repository/LinkageRepositoryTest.java [hunk 5]
```diff
@@ -171,6 +171,7 @@ public void getLinkedMarkersFromGene() {
 
     @Test
     public void getLinkedMarkersFromSSLP() {
+
```
Comment (rtaylorzfin) [inline test/org/zfin/mapping/repository/LinkageRepositoryTest.java:174]: remove blank lines

**Review by cmpich — APPROVED** [review]
Great work.

**Pull request merged**